src/bpftrace.cpp

```diff
@@ -2124,4 +2124,26 @@ void BPFtrace::sort_by_key(std::vector<SizedType> key_args,
   }
 }
 
+std::string BPFtrace::get_string_literal(const ast::Expression *expr) const
```

Comment:
bit of a nit: I don't think this belongs in the bpfrace class. bpftrace already does way to much and this is very specific to the ast.  `ast/ast.h` might be a better place for it

Reply:
I thought the same, but then I realized that it is specific to BPFtrace class since the class holds values of positional parameters that this function needs. I can move it to `ast/ast.h`, but then I'd have to add a reference to bpftrace as a new parameter (which I don't like too much as it breaks encapsulation).

Reply:
Hmm fair point. Neither of the options are great. Passing "bpftrace" along feels a bit less wrong to me as its a global state we pass along everywhere anyway (although a name like context might be better). But it's easy to refactor this later on.

@danobi  any strong feelings?